spec/miscellaneous/mailers/user_mailer_spec.rb

```diff
@@ -285,4 +285,68 @@
       end
     end
   end
+
+  describe "invite_increase_notification" do
+    let!(:user) { create(:user) }
+
+    context "when 1 invitation is issued" do
+      let(:count) { 1 }
+
+      subject(:email) { UserMailer.invite_increase_notification(user.id, count).deliver }
+
+      # Test the headers
+      it_behaves_like "an email with a valid sender"
+
+      it "has the correct subject line" do
+        expect(email.subject).to eq("[#{ArchiveConfig.APP_SHORT_NAME}] New Invitations")
+      end
+
+      # Test both body contents
+      it_behaves_like "a multipart email"
+
+      it_behaves_like "a translated email"
+
+      describe "HTML version" do
+        it "has the correct content" do
+          expect(email).to have_html_part_content("you have #{count} new invitation")
+        end
+      end
+
+      describe "text version" do
+        it "has the correct content" do
+          expect(email).to have_text_part_content("you have #{count} new invitation")
+        end
+      end
+    end
+
+    context "when multiple invitations are issued" do
+      let(:count) { 5 }
+
+      subject(:email) { UserMailer.invite_increase_notification(user.id, count).deliver }
+
+      # Test the headers
+      it_behaves_like "an email with a valid sender"
+
+      it "has the correct subject line" do
+        expect(email.subject).to eq("[#{ArchiveConfig.APP_SHORT_NAME}] New Invitations")
+      end
+
+      # Test both body contents
+      it_behaves_like "a multipart email"
+
+      it_behaves_like "a translated email"
+
+      describe "HTML version" do
+         it "has the correct content" do
+          expect(email).to have_html_part_content("you have #{count} new invitations")
+        end
```

Comment:
We should fix this.

Reply:
I'm not sure why this didn't get dismissed... it definitely got fixed when I dealt with `it "has the correct content" do` being misaligned (it was indented 3 spaces instead of 2). Here are the lines copied with their leading spaces:

```
        it "has the correct content" do
        end
```

Reply:
Looks good, we'll just manually dismiss.